src/components/ebay-select/index.js

```diff
@@ -120,7 +120,6 @@ function setSelectedOption(optionValue) {
         } else {
             option.selected = false;
         }
-        this.update();
```

Comment:
Are you sure? If my memory serves me correctly (and it's faulty, TBH) I remember it was needed for the component to receive the updated state immediately, and not wait for the lifecycle to complete.

Reply:
https://github.com/eBay/ebayui-core/pull/209/files#diff-fed042b9a0d6b10a5ae8d5f32fa7154aR122

Reply:
yes, since .map creates new options array.

Reply:
We shouldn’t really be using update directly since it causes the component to update immediately. Typically you want `forceUpdate` which tells Marko that this component is dirty and needs to update but allows Marko to schedule the actual update.

Also we don’t need to signal to Marko that we need to update in this case because the options array is rebuilt every time and the reference equality difference will be enough to make Marko update.

This PR looks good to me.